garak/resources/tap/generator_utils.py

```diff
@@ -21,6 +21,7 @@
 }
 
 
+# replace with __init__ version using _config
```

Comment:
Is this an open todo?

Reply:
There's a lot of refactoring to do in this module.

Reply:
I believe this is an open TODO, yes.